### ISSUE TITLE
Added fr (French) translation

### ISSUE THREAD

**Issue by theofruitrouge** [AdAway/AdAway#888]
I have updated the French translation of your app. `fr` (*French*). Here is the xml:
```xml
<!-- File "strings_preferences.xml" -->
<?xml version="1.0" encoding="utf-8"?>
<resources>

    <string name="pref_update_check_key" translate="false">updateCheck</string>


    <string name="pref_never_reboot_key" translate="false">neverReboot</string>


    <string name="pref_enable_ipv6_key" translate="false">enableIpv6</string>


    <string name="pref_update_check_daily_key" translate="false">updateCheckDaily</string>


    <string name="pref_automatic_update_daily_key" translate="false">automaticUpdateDaily</string>


    <string name="pref_update_only_on_wifi_key" translate="false">updateOnlyOnWifi</string>


    <string name="pref_whitelist_rules_key" translate="false">whitelistRules</string>


    <string name="pref_redirection_rules_key" translate="false">redirectionRules</string>


    <string name="pref_redirection_ip_key" translate="false">redirectionIP</string>


    <string name="pref_apply_method_key" translate="false">applyMethod</string>

    <string-array name="pref_apply_method_entries_values" translate="false">
        <item>writeToSystem</item>
        <item>writeToData</item>
        <item>writeToDataData</item>
        <item>customTarget</item>
    </string-array>

    <string name="pref_apply_method_def" translate="false">writeToSystem</string>
    <string name="pref_custom_target_key" translate="false">customTarget</string>


    <string name="pref_webserver_enabled_key" translate="false">webserverEnabled</string>


    <string name="pref_webserver_on_boot_key" translate="false">webserverOnBoot</string>


    <string name="pref_webserver_version_key" translate="false">webserverVersion</string>


    <string name="pref_tcpdump_version_key" translate="false">tcpdumpVersion</string>


    <string name="pref_enable_debug_key" translate="false">debugEnabled</string>


    <string name="pref_enable_systemless_key">enableSystemless</string>

    <string name="pref_enable_systemless_def">faux</string>

</resources>

<!-- File "strings_static.xml" -->
<?xml version="1.0" encoding="utf-8"?>
<resources>

    <string name="about_url" translate="false">http://code.google.com/p/ad-away/</string>
    <string name="no_root_url" translate="false">http://www.cyanogenmod.org</string>
    <string name="hosts_add_dialog_input" translate="false">http://</string>

</resources>

<!-- File "strings.xml" -->
<?xml version="1.0" encoding="utf-8"?>
<resources>

    <string name="app_name">AdAway</string>

    <!-- Buttons -->
    <string name="button_exit">Quitter</string>
    <string name="button_apply">Télécharger les fichiers et appliquer le blocage des publicités</string>
    <string name="button_close">Fermer</string>
    <string name="button_yes">Oui</string>
    <string name="button_no">Non</string>
    <string name="button_revert">Désactiver le blocage des publicités</string>
    <string name="button_add">Ajouter</string>
    <string name="button_cancel">Annuler</string>
    <string name="button_apply_text">Tous les fichiers « hosts » provenant des sources définies sont téléchargés, fusionnés avec vos propres listes et installés en un seul fichier « hosts » sur votre système.</string>
    <string name="button_revert_text">Restaure le fichier « hosts » par défaut sur votre système.</string>
    <string name="button_save">Enregistrer</string>
    <string name="button_webserver_toggle_checked">Le serveur Web tourne</string>
    <string name="button_webserver_toggle_unchecked">Le serveur Web est arrêté</string>
    <string name="button_webserver_toggle_text">Démarrer ou arrêter le serveur Web sur « localhost » pour répondre aux requêtes pour des noms d\'hôtes bloqués.</string>
    <string name="button_help">Aide</string>
    <string name="button_tcpdump_text">Tcpdump est un outil pour surveiller les requêtes DNS et les enregistrer dans un fichier journal. Vous pouvez le démarrer en arrière-plan, lancer des applications qui affichent des publicités, et analyser ensuite les requêtes DNS d\'après le fichier journal. Le serveur publicitaire potentiel peut alors être ajouté à votre liste noire.</string>
    <string name="button_tcpdump_toggle_checked">Tcpdump tourne !</string>
    <string name="button_tcpdump_toggle_unchecked">Tcpdump est arrêté</string>
    <string name="button_tcpdump_open">Ouvrir le fichier journal</string>
    <string name="button_tcpdump_delete">Supprimer le fichier journal</string>

    <!-- Checkbox List -->
    <string name="checkbox_list_add_dialog_title">Ajouter</string>
    <string name="checkbox_list_empty">Liste vide</string>
    <string name="checkbox_list_empty_text">Vous pouvez ajouter des éléments depuis la barre d\'action.</string>
    <string name="checkbox_list_context_delete">Supprimer</string>
    <string name="checkbox_list_footer">Après l\'ajout de nouveaux éléments à cette liste, AdAway doit être appliqué à nouveau.</string>
    <string name="checkbox_list_edit_dialog_title">Modifier</string>
    <string name="checkbox_list_context_title">Modifier l\'entrée</string>
    <string name="checkbox_list_context_edit">Modifier</string>

    <!-- Your Lists -->
    <string name="lists_tab_whitelist">Liste blanche</string>
    <string name="lists_tab_blacklist">Liste noire</string>
    <string name="lists_tab_blacklist_short">Noire</string>
    <string name="lists_tab_whitelist_short">Blanche</string>
    <string name="lists_tab_redirection_list_short">Redirections</string>
    <string name="lists_tab_redirection_list">Liste des redirections</string>
    <string name="list_dialog_url">URL :</string>
    <string name="list_dialog_hostname">Nom d\'hôte :</string>
    <string name="list_dialog_whitelist_hostname">Nom d\'hôte :\n
    (Les métacaractères * et ? sont autorisés)</string>
    <string name="list_dialog_ip">IP (IPv4 ou IPv6) :</string>
    <string name="import_dialog">Importation...</string>
    <string name="export_dialog">Exportation...</string>
    <string name="export_success">Toutes les entrées ont été exportées vers un fichier « adaway-export » sur votre carte SD.</string>

    <!-- Status Notifications -->
    <string name="status_checking">Vérification de mise à jour...</string>
    <string name="status_reverting">Retour arrière...</string>
    <string name="status_reverting_subtitle">Création d\'un fichier « hosts » standard</string>
    <string name="status_checking_subtitle">Vérifier si une mise à jour est proposée</string>
    <string name="status_update_available">Mise à jour proposée</string>
    <string name="status_update_available_subtitle">Nouveaux fichiers « hosts » proposés</string>
    <string name="status_no_connection">Aucune connexion Internet</string>
    <string name="status_no_connection_subtitle">Aucune connexion Internet</string>
    <string name="status_download_fail">Sources non disponibles</string>
    <string name="status_download_fail_subtitle_new">Aucune source «hosts » n\'est joignable !</string>
    <string name="status_enabled">Activé</string>
    <string name="status_enabled_subtitle">Fichiers « hosts »  les plus récents activés</string>
    <string name="status_disabled">Désactivé</string>
    <string name="status_disabled_subtitle">Désactivé</string>

    <!-- Apply and Revert Dialogs -->
    <string name="apply_symlink_missing">Le lien symbolique de votre cible vers /system/etc/hosts est absent ou incorrect ! AdAway ne fonctionnera pas sans un lien vers le bon fichier.\n
    \n
    Voulez-vous essayer de créer un lien symbolique ?</string>
    <string name="apply_symlink_missing_title">Lien symbolique manquant</string>
    <string name="apply_not_enough_space">Manque d\'espace sur la partition !\n
    Essayez de modifier le fichier « hosts » cible dans les préférences par /data/data/hosts.</string>
    <string name="apply_not_enough_space_title">Manque d\'espace</string>
    <string name="apply_private_file_fail">Impossible de lire le fichier téléchargé.</string>
    <string name="apply_private_file_fail_title">Échec de fichier privé</string>
    <string name="apply_remount_fail">Échec de remontage de la partition en lecture/écriture !</string>
    <string name="apply_remount_fail_title">Échec de remontage</string>
    <string name="apply_copy_fail">Échec de copie du fichier « hosts » !</string>
    <string name="apply_copy_fail_title">Échec de copie</string>
    <string name="apply_success_subtitle">Sources « hosts » mises en place :</string>
    <string name="apply_success_title">Mise à en place réussie</string>
    <string name="apply_success_dialog">Mise en place réussie.\n
    Il pourrait être nécessaire de redémarrer Android pour que les modifications prennent effet.\n
    \n
    Voulez-vous redémarrer ?\n
    (Pour éviter toute perte de données, assurez-vous qu\'aucune appli n\'utilise la carte SD actuellement !)</string>
    <string name="apply_symlink_successful">Le lien symbolique a été créé avec succès.\n
    Il pourrait être nécessaire de redémarrer Android pour que les modifications prennent effet.\n
    \n
    Voulez-vous redémarrer ? (Pour éviter toute perte de données, assurez-vous qu\'aucune application n\'utilise la carte SD actuellement !)</string>
    <string name="apply_symlink_successful_title">Création du lien symbolique réussi</string>
    <string name="apply_symlink_fail">Le lien symbolique n\'a pas pu être créé par Android.\n
    Ceci échoue la plupart du temps à cause d\'une fonction appelée S-ON sur les téléphones HTC !\n
    \n
    Une solution est de démarrer votre téléphone en mode récupération et d\'y créer un lien symbolique avec « ln-s /data/data/hosts/system/etc/hosts ».\n
    \n
    Si cela ne fonctionne pas, rechercher S-OFF et votre téléphone sur Internet.</string>
    <string name="apply_symlink_fail_title">Échec de la création du lien symbolique</string>
    <string name="apply_help">Veuillez lire l\'aide pour plus d\'informations, .</string>
    <string name="apply_apn_proxy">Un mandataire APN est configuré sur votre Android !\n
    AdAway ne fonctionnera pas de manière fiable sur les réseaux mobiles comme le 3G. Vous pouvez désactiver ce mandataire en allant à l\'APN choisi (sur Android 4 : Sans fil et réseaux -> Paramètres supp. -> Réseaux Mobile -> Nom des point d\'accès) et en enlevant la valeur dans le champ « proxy »</string>
    <string name="apply_apn_proxy_title">Mandataire APN configuré !</string>
    <string name="no_connection">La connexion à Internet me fonctionne pas.</string>
    <string name="no_connection_title">Aucune connexion</string>
    <string name="download_dialog">Téléchargement...</string>
    <string name="apply_dialog">Mise en place...</string>
    <string name="apply_dialog_lists">Mise en place des liste noire et blanche</string>
    <string name="apply_dialog_hostnames">Analyse et fusion des fichiers « hosts »</string>
    <string name="apply_dialog_hosts">Création du fichier « hosts »</string>
    <string name="apply_dialog_apply">Mise en place du fichier « hosts »</string>
    <string name="apply_fail">Le fichier « hosts » n\'a pas pu être appliqué à votre système !\n
    Essayez de changer le fichier « hosts » cible dans les préférences par /data/data/hosts.</string>
    <string name="apply_fail_title">Échec lors de la mise en place</string>
    <string name="revert_successful">Vous êtes revenu au fichier « hosts » par défaut.\n
    Il pourrait peut-être nécessaire de redémarrer Android pour que les changements prennent effet.\n
    \n
    Voulez-vous redémarrer ?\n
    (Afin d\'éviter la perte de données, assurez-vous qu\'aucune application n\'utilise la carte SD actuellement !)</string>
    <string name="revert_successful_title">Retour arrière réussi</string>
    <string name="revert_problem">Le retour arrière a échoué pour des raisons inconnues.</string>
    <string name="revert_problem_title">Échec du retour arrière !</string>
    <string name="download_fail_dialog">Aucune de vos sources d\'« hosts » activées n\'est disponible. Êtes-vous correctement connecté à internet ?</string>
    <string name="download_fail_title">Échec de téléchargement</string>
    <string name="no_private_file">Le fichier privé ne peut pas être créé.</string>
    <string name="no_private_file_title">Échec de création du fichier privé</string>
    <string name="enable_systemless_successful">Vous avez activé le mode systemless.\n
  Il est possible que vous deviez redémarer Android pour que les changements prennent effet.\n
  \n
  Voulez-vous redémarer ?\n
  (Pour éviter les pertes de données, assurez-vous qu\'aucune application n\'utilise la carte SD en ce moment !)</string>
    <string name="enable_systemless_successful_title">Activation réussie</string>
    <string name="disable_systemless_successful">Vous avez désactivé le mode sans système.\n
    Il peut être nécessaire de redémarrer Android pour que les changements prennent effet.\n
    \n
    Voulez-vous redémarrer?\n
    (Pour prévenir la perte de données, assurez-vous qu\'aucune autre application n’utilise la carte SD en ce moment !)</string>
    <string name="disable_systemless_successful_title">Désactivation réussie</string>

    <!-- Other Dialogs -->
    <string name="no_root_description">Soit le fichier binaire « su » est introuvable, soit vous n\'avez pas donné l\'accès root à AdAway.\n
    \n
    Cela peut arriver quand votre appareil n\'est pas enraciné\n
    Vous trouverez des informations sur comment enraciner votre appareil sur www.cyanogenmod.org ou d\'autres sites Web dédiés à Android.</string>
    <string name="no_root_title">Un Android rooté est exigé</string>
    <string name="no_url">Ce n\'est pas une URL valide !</string>
    <string name="no_url_title">URL mal formatée</string>
    <string name="hosts_add_dialog">Ajouter une URL au fichier « hosts » :</string>
    <string name="no_hostname">Ce n\'est pas un nom d\'hôte valide !</string>
    <string name="no_hostname_title">Nom d\'hôte mal formaté</string>
    <string name="no_ip">Ce n\'est pas une IP valide !</string>
    <string name="no_ip_title">IP mal formatéee</string>
    <string name="reboot_never_reboot">Ne jamais redémarrer et ne plus demander à l\'avenir !</string>
    <string name="loading">Chargement...</string>

    <!-- Menu entries -->
    <string name="menu_refresh">Rafraîchir</string>
    <string name="menu_add">Ajouter</string>
    <string name="menu_scan_adware">Recherche de pubiciels</string>
    <string name="menu_hosts_sources">Sources d\'hôtes</string>
    <string name="menu_tcpdump">Journaliser les requêtes DNS</string>
    <string name="menu_help">Aide</string>
    <string name="menu_lists">Vos listes</string>
    <string name="menu_show_hosts_file">Ouvrir le fichier « hosts »</string>
    <string name="menu_import">Importer toutes les listes</string>
    <string name="menu_export">Exporter toutes les listes</string>
    <string name="menu_preferences">Préférences</string>

    <!-- Help -->
    <string name="help_tab_about">À propos de</string>
    <string name="help_tab_donate">Faire un don</string>
    <string name="help_tab_s_on_s_off">S-ON/S-OFF</string>
    <string name="help_tab_faq">FAQ</string>
    <string name="help_tab_problems">Problèmes</string>
    <string name="help_tab_changelog">Journal des changements</string>
    <string name="help_about_version">Version :</string>
    <string name="help_donation_paypal_item">Faire un don à AdAway</string>
    <string-array name="help_donation_google_catalog_values">
        <item>1 euro</item>
        <item>2 euros</item>
        <item>3 euros</item>
        <item>5 euros</item>
        <item>8 euros</item>
        <item>13 euros</item>
    </string-array>

    <!-- Scan Adware -->
    <string name="scan_adware_description">Vous pouvez rechercher ici les mauvais pubiciels qui ne peuvent être bloqués par AdAWay. Ces applis utilisent par exemple des notifications intempestives « airpush » qui s\'affichent même si l\'appli ne tourne pas et peuvent aussi changer votre sonnerie. La seule contre-mesure est de désinstaller ces applis en cliquant sur les éléments listés !</string>
    <string name="scan_adware_start">Démarrer l\'analyse</string>
    <string name="scan_adware_empty">Aucun mauvais pubiciel trouvé !</string>

    <!-- Hosts Sources -->
    <string name="hosts_last_modified_local">Local :</string>
    <string name="hosts_last_modified_online">En ligne :</string>
    <string name="hosts_not_available">non disponible</string>

    <!-- Other -->
    <string name="app_subtitle">Blocage de publicités</string>
    <string name="no_text_editor_title">Aucun éditeur de texte n\'est installé</string>
    <string name="no_text_editor">Aucun éditeur de texte n\'a été trouvé pour ouvrir le fichier « hosts ». Vous pouvez installer Jota ou d\'autres éditeurs de texte pour ce faire.\n
    \n
    Voulez-vous installer Jota ?</string>
    <string name="no_file_manager_title">Aucun gestionnaire de fichiers n\'est installé</string>
    <string name="no_file_manager">Aucun gestionnaire de fichier n\'a trouvé pour ouvrir les fichiers. Vous pouvez installer OI File Manager ou d\'autres gestionnaires de fichiers pour ce faire.\n
    \n
    Voulez-vous installer OI File Manager ?</string>

    <!-- Tcpdump -->
    <string name="toast_tcpdump_log_deleted">Le fichier journal a été supprimé !</string>
    <string name="toast_tcpdump_added_to_blacklist">Le nom d\'hôte a été ajouté à votre liste noire !</string>
    <string name="toast_tcpdump_added_to_whitelist">Le nom d\'hôte a été ajouté à votre liste blanche !</string>
    <string name="tcpdump_log_context_blacklist">Ajouter à votre liste noire</string>
    <string name="tcpdump_log_context_whitelist">Ajouter à votre liste blanche</string>
    <string name="tcpdump_log_context_browser">Ouvrir dans le navigateur</string>
    <string name="tcpdump_log_empty">Le fichier journal est vide !</string>
    <string name="tcpdump_log_footer">Appuyer longuement sur un nom d\'hôte pour l\'ajouter à votre liste noire ou blanche !</string>

    <!-- Preferences -->
    <string name="pref_download">Préférences générales</string>
    <string name="pref_hosts">Fichier « hosts »</string>
    <string name="pref_redirection_ip">IP de redirection</string>
    <string name="pref_update_check">Vérifier les mises à jour</string>
    <string name="pref_apply_method">Fichier « hosts » cible</string>
    <string name="pref_apply_method_summary">Avant de modifier ce paramètre, lire l\'aide concernant cette fonction.</string>
    <string name="pref_redirection_ip_summary">Tous les noms d\'hôtes bloqués sont redirigés vers cette adresse IP.</string>
    <string name="pref_header_preferences">Préférences</string>
    <string name="pref_header_preferences_summary">Préférences générales d\'AdAway</string>
    <string name="pref_never_reboot">Cacher la fenêtre de redémarrage</string>
    <string name="pref_never_reboot_summary">Masque la demande de redémarrage d\'Android après une mise à en place réussie</string>
    <string name="pref_webserver_on_boot">Lancer le serveur Web au démarrage</string>
    <string name="pref_webserver_on_boot_summary">Lance le serveur Web au démarrage, à l\'écoute des adresses IP locales</string>
    <string name="pref_webserver_enabled">Activer le serveur Web</string>
    <string name="pref_webserver_enabled_summary">Active le serveur Web à l\'écoute d\'adresses IP locales pour répondre aux requêtes vers des noms d\'hôtes bloqués</string>
    <string name="pref_webserver">Serveur Web local</string>
    <string name="pref_custom_target">Cible personnalisée</string>
    <string name="pref_custom_target_summary">Utilisé uniquement si le fichier « hosts » cible est personnalisé</string>

    <string-array name="pref_apply_method_entries">
        <item>/system/etc/hosts</item>
        <item>/data/hosts</item>
        <item>/data/data/hosts</item>
        <item>Cible personnalisée</item>
    </string-array>

    <string name="pref_update_check_summary">Vérifier les mises à jour lors du démarrage d\'AdAway</string>
    <string name="pref_sdcard_problem">Cette fonction ne fonctionnera pas correctement si AdAway est installé sur une carte SD !</string>
    <string name="pref_background">Services d\'arrière-plan</string>
    <string name="pref_update_check_daily">Vérifier les mises à jour quotidiennement</string>
    <string name="pref_update_check_daily_summary">Vérifier les mises à jour quotidiennement en arrière-plan avec des notifications</string>
    <string name="pref_automatic_update_daily">Mise à jour automatique</string>
    <string name="pref_automatic_update_daily_summary">Mettre à jour automatiquement en arrière-plan si des mises à jours sont proposées</string>
    <string name="pref_update_only_on_wifi">Mise à jour Wi-Fi seulement</string>
    <string name="pref_update_only_on_wifi_summary">N\'exécuter les mises à jour en arrière-plan que si connecté à un réseau Wi-Fi.</string>
    <string name="pref_whitelist_rules">Permettre la mise en liste blanche</string>
    <string name="pref_whitelist_rules_summary">Permettre les entrées de liste blanche des sources « hosts »</string>
    <string name="pref_redirection_rules">Permettre les redirections</string>
    <string name="pref_redirection_rules_summary">CECI POSE UN RISQUE DE SÉCURITÉ ! Permettre les entrées de redirection de sources « hosts ».</string>
    <string name="pref_debug">Debogage</string>
    <string name="pref_enable_debug">Journalisation du débogage</string>
    <string name="pref_enable_debug_summary">Vous devez redémarrer AdAway pour que ce paramètre prenne effet.</string>
    <string name="pref_enable_ipv6">Activer IPv6</string>
    <string name="pref_enable_ipv6_summary">Toutes les entrées du fichier host auront une entrée ::1 (IPv6 localhost) correspondante.</string>
    <string name="pref_enable_systemless_summary">Empêcher AdAway de modifier la partition système (SuperSU sans système est exigé pour une installation automatique)</string>
    <string name="pref_enable_systemless">Activer le mode sans système</string>

</resources>


```

---

*Translation made with [Stringlate](https://github.com/LonamiWebs/Stringlate).*

**Comment by Vankog** [comment]
@theofruitrouge 
Alright, I finally got the permissions and time to get things in order.  
Thank you so much for your effort!  
  
Unfortunately, as it turned out, we cannot merge translations via Github directly. Translations are managed via the **transifex.com** website! (and Transifex alone, that is).  
I really tried to find a solution, but I am afraid we just can't blindly merge/upload the files. Strings change constantly and there are already hundreds of competing translation pieces on Transifex. It is an organic growth by dozens of people worldwide.  

Therefore, you can integrate your current translation to the service yourself by following the steps below.  
I am really sorry! But again: This is due to some major synchronization issues if not followed.  We have to consolidate many contributions by translators and sync them up to the latest state. That is just not possible via Github.  

1. Please go to **https://www.transifex.com/free-software-for-android/adaway/**
1. Login or create a new account (you can conveniently login via Github as well)
1. Enroll into the language you want to contribute to or even submit a request for a new language.
   * Please keep in mind that we want to stick to the basic languages wehere possible (e.g. `sr` for Serbian).  
   Please refrain to request regional localizations (like `sr_RS`).
1. In your language section, you can browse all available resources and start **translating strings right in your browser**.
   * You don't have to download anything. Just click "Translate". The downloads are meant for more advanced use cases.
1. Make sure to have an eye on the **"Suggestions", "History", "Context" and "Glossary" tabs** on every entry. 
   * Mind translation efforts that were already done or suggested.  
   Basically, »*stand on the shoulders of giants*« where possible.
   * Sometimes source strings change only marginally. You can easily **recover their previous translations** by looking at the "Suggestions" tab. Just make sure they really fit the new source text.
   * Help us validate translations by **reviewing others**. 
1. Some strings contain **placeholders** - like for HTML tags or numbers.  
You can click on them to add them or use keyboard shortcuts (see the settings for an overview).
1. You can point out issues to the managers via the **"Comments" tab** or just discuss some other matters.
1. Don't forget to save your work! 
1. For more information about how to use Transifex, see https://docs.transifex.com/


Thank you very much! This helps a lot!
